lib/e_chronicler_web/router.ex

```diff
@@ -21,6 +21,9 @@ defmodule EChroniclerWeb.Router do
     get "/journal_entry/new", JournalEntryController, :new
     get "/journal_entry/:id", JournalEntryController, :show
     post "/journal_entry", JournalEntryController, :create
+    get "/journal_entry/:id/edit", JournalEntryController, :edit
+    patch "/journal_entry/:id", JournalEntryController, :update
+    put "/journal_entry/:id", JournalEntryController, :update
```

Comment:
Might be worth reading up on `PUT` vs `PATCH` and figure out which method is the one you need. (This will give you a leg up on your next project, the HTTP server, too!)

Reply:
Looks like `PUT` since we modify the entire resource.